README.md

```diff
@@ -1,9 +1,10 @@
 # dss-plugin-tesseract-ocr
-Plugin for optical character recognition (OCR) in python using the tesseract engine
+Plugin for text extraction and optical character recognition (OCR) in python.
 
-The plugin has four components (three recipes and a notebook template):
+The plugin has five components (four recipes and a notebook template):
+- OCR - Text Extraction recipe: extract text from images using either the tesseract or the easyOcr engine and output a dataset with a filename column and a text column. This recipe has parameters to recombine multi-page PDF (for instance those that were split into multiple images in the Image Conversion recipe) into a single text row, to specify the language used in tesseract, and to remove special charaters from the extracted text. It takes as input a folder of PDF/JPG/JPEG/PNG/TIFF files (ideally the output of the Image Conversion recipe or of the Image Processing recipe).
```

Comment:
I would simply rename it to `OCR recipe` or `Optical Character Recognition recipe` to avoid confusion with the `Text Extraction recipe`